### PR TITLE
[All Hosts] (auth) SSO not supported in 'Edit in Teams'

### PR DESCRIPTION
**Pull request by Rick-Kirkham**
Partially fixes [4087](https://github.com/OfficeDev/office-js-docs-pr/issues/4087)

There will be an identically named PR in the conceptual repo too.

### CODE REVIEW

File: docs/requirement-sets/common/identity-api-requirement-sets.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
 ---
 title: Identity API requirement sets
 description: Identity API requirement set information for Office Add-ins.
-ms.date: 09/28/2022
+ms.date: 07/07/2023
 ms.topic: overview
+ms.prod: non-product-specific
+
```
Comment (ElizabethSamuel-MSFT) [inline docs/requirement-sets/common/identity-api-requirement-sets.md:7]: No more need for ms.prod. We now use ms.service and it's handled in the [docfx.json](https://github.com/OfficeDev/office-js-docs-reference/blob/main/docs/docfx.json) unless you want to override its value at the file level. The default ms.service value for this file found in the docfx is microsoft-365.

```suggestion
```

File: docs/requirement-sets/common/identity-api-requirement-sets.md [hunk 2]
```diff
@@ -19,15 +21,17 @@ Office Add-ins run across multiple versions of Office. The following table lists
 > \* Currently, the IdentityAPI 1.3 requirement set is supported in Office on the web only for documents that are opened from Microsoft SharePoint Online and OneDrive.
 
 > [!IMPORTANT]
-> In Outlook, the Identity API requirement set isn't supported if the add-in is loaded in an Outlook.com or Gmail mailbox.
+>
+> - Regardless of the platform, the Identity API requirement set isn't supported if the Office document is opened from the **Files** tab of a Teams channel using the **Edit in Teams** option on the **Open** drop down menu.
```
Comment (ElizabethSamuel-MSFT) [inline docs/requirement-sets/common/identity-api-requirement-sets.md:25]: Per [style guide](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=28800), this should be dropdown

```suggestion
> - Regardless of the platform, the Identity API requirement set isn't supported if the Office document is opened from the **Files** tab of a Teams channel using the **Edit in Teams** option on the **Open** dropdown menu.
```

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

**Comment by Rick-Kirkham** [comment]
@AJ1062910 Angular is still supported and you can still make add-ins with Angular. However, our documentation and tools teams don't have the capacity to support an Angular option in Yo Office anymore. We simply don't have the time to maintain it.

**Comment by Rick-Kirkham** [comment]
Regardless of how you create your project, you will have security issues. New security issues are discovered all the time. There is no way to avoid security issues. Creating an add-in with Yo Office doesn't protect you from security issues. There will still be new ones discovered after you have created your project. 

If you want to create an add-in with Angular, you can do so, but you will need to create the project from scratch. You can no longer create an add-in project that uses Angular from the Yo Office tool.

**Comment by Rick-Kirkham** [comment]
@AJ1062910 I'm afraid not. The best thing to do when you have questions is to raise them in one of these places:

- [Microsoft Q&A](https://learn.microsoft.com/en-us/answers/questions/) (tag your question "Office Development"
- [Stack Overflow](https://stackoverflow.com/) (tag your question "office-js".

**Pull request merged**